class-unlist-posts-admin.php

```diff
@@ -92,10 +92,10 @@ function metabox_render( $post ) {
 		?>
 		<p>
 			<label class="checkbox-inline">
-				<input name="unlist_posts" type="checkbox" <?php echo esc_attr( $checked ); ?> value=""><?php _e( 'Unlist this post?', 'unlist-posts' ); ?>
+				<input name="unlist_posts" type="checkbox" <?php echo esc_attr( $checked ); ?> value=""><?php echo esc_html( __( 'Unlist this post?', 'unlist-posts' ) ); ?>
```

Comment:
You can use the **_esc_html_e_** function here.

Reply:
Updated